Makefile

```diff
@@ -175,7 +125,6 @@ info:
 	@echo
 
 
-.PHONY: build
```

Comment:
How come the .PHONY's are removed? I thought they were to tell make to not expect a file by this name to be created as a product of this target

Reply:
Yeah technically there is just one needed but I removed them in the process of debugging the make file. I should add them back in.